File: Commande.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Commande
 *
 * @ORM\Table(name="commande", indexes={@ORM\Index(name="idcart", columns={"idcart"})})
 * @ORM\Entity(repositoryClass="App\Repository\CommandeRepository")
 */
class Commande
{
    /**
     * @var int
     *
     * @ORM\Column(name="commandeId", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $commandeid;



    /**
     * @var string
     *
     * @ORM\Column(name="nom", type="string", length=225, nullable=false)
     */
    private $nom;

    /**
     * @var string
     *
     * @ORM\Column(name="prenom", type="string", length=225, nullable=false)
     */
    private $prenom;

    /**
     * @var string
     *
     * @ORM\Column(name="email", type="string", length=225, nullable=false)
     */
    private $email;

    /**
     * @var string
     *
     * @ORM\Column(name="adresse", type="string", length=225, nullable=false)
     */
    private $adresse;

    /**
     * @var int
     *
     * @ORM\Column(name="numTelephone", type="integer", nullable=false)
     */
    private $numtelephone;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="datetime", nullable=false)
     */
    private $date;

    /**
     * @var int
     *
     * @ORM\Column(name="totalCost", type="integer", nullable=false)
     */
    private $totalcost;

    public function getCommandeid(): ?int
    {
        return $this->commandeid;
    }




    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->prenom;
    }

    public function setPrenom(string $prenom): self
    {
        $this->prenom = $prenom;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(string $email): self
    {
        $this->email = $email;

        return $this;
    }

    public function getAdresse(): ?string
    {
        return $this->adresse;
    }

    public function setAdresse(string $adresse): self
    {
        $this->adresse = $adresse;

        return $this;
    }

    public function getNumtelephone(): ?int
    {
        return $this->numtelephone;
    }

    public function setNumtelephone(int $numtelephone): self
    {
        $this->numtelephone = $numtelephone;

        return $this;
    }

    public function getDate(): ?\DateTimeInterface
    {
        return $this->date;
    }

    public function setDate(\DateTimeInterface $date): self
    {
        $this->date = $date;

        return $this;
    }

    public function getTotalcost(): ?int
    {
        return $this->totalcost;
    }

    public function setTotalcost(int $totalcost): self
    {
        $this->totalcost = $totalcost;

        return $this;
    }


}

File: src/com/mycompany/myap/entities/Commande.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myap.entities;

import java.util.Date;

/**
 *
 * @author azizm
 */
public class Commande {
   
	
	private int commandeId;
        private int idcart ;
        private String nom ; 
        private String prenom ;
        private String email;
        private String Adresse; 
        private int numTelephone;
	private String Date;
	private int totalCost;

    public Commande() {
    }

    @Override
    public String toString() {
        return "Commande{" + "commandeId=" + commandeId + ", idcart=" + idcart + ", nom=" + nom + ", prenom=" + prenom + ", email=" + email + ", Adresse=" + Adresse + ", numTelephone=" + numTelephone + ", Date=" + Date + ", totalCost=" + totalCost + '}';
    }

    public Commande(String nom, String prenom, String email, String Adresse, int numTelephone, String Date, int totalCost) {
        this.nom = nom;
        this.prenom = prenom;
        this.email = email;
        this.Adresse = Adresse;
        this.numTelephone = numTelephone;
        this.Date = Date;
        this.totalCost = totalCost;
    }

    public Commande(int idcart, String nom, String prenom, String email, String Adresse, int numTelephone, String Date, int totalCost) {
        this.idcart = idcart;
        this.nom = nom;
        this.prenom = prenom;
        this.email = email;
        this.Adresse = Adresse;
        this.numTelephone = numTelephone;
        this.Date = Date;
        this.totalCost = totalCost;
    }

    public Commande(int commandeId, int idcart, String nom, String prenom, String email, String Adresse, int numTelephone, String Date, int totalCost) {
        this.commandeId = commandeId;
        this.idcart = idcart;
        this.nom = nom;
        this.prenom = prenom;
        this.email = email;
        this.Adresse = Adresse;
        this.numTelephone = numTelephone;
        this.Date = Date;
        this.totalCost = totalCost;
    }

    public int getCommandeId() {
        return commandeId;
    }

    public void setCommandeId(int commandeId) {
        this.commandeId = commandeId;
    }

    public int getIdcart() {
        return idcart;
    }

    public void setIdcart(int idcart) {
        this.idcart = idcart;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public String getAdresse() {
        return Adresse;
    }

    public void setAdresse(String Adresse) {
        this.Adresse = Adresse;
    }

    public int getNumTelephone() {
        return numTelephone;
    }

    public void setNumTelephone(int numTelephone) {
        this.numTelephone = numTelephone;
    }

    public String getDate() {
        return Date;
    }

    public void setDate(String Date) {
        this.Date = Date;
    }

    public int getTotalCost() {
        return totalCost;
    }

    public void setTotalCost(int totalCost) {
        this.totalCost = totalCost;
    }

   
}